[2025-08-06T16:04:45.986+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-06T16:04:45.993+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_s3_metadata_dag.extract_holiday_events manual__2025-08-06T16:04:44.805420+00:00 [queued]>
[2025-08-06T16:04:45.997+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_s3_metadata_dag.extract_holiday_events manual__2025-08-06T16:04:44.805420+00:00 [queued]>
[2025-08-06T16:04:45.998+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-06T16:04:46.012+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_holiday_events> on 2025-08-06 16:04:44.805420+00:00
[2025-08-06T16:04:46.042+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=577) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-06T16:04:46.044+0000] {standard_task_runner.py:63} INFO - Started process 587 to run task
[2025-08-06T16:04:46.044+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_s3_metadata_dag', 'extract_holiday_events', 'manual__2025-08-06T16:04:44.805420+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/extract/mysql_to_s3_dag.py', '--cfg-path', '/tmp/tmp_5l1elyy']
[2025-08-06T16:04:46.048+0000] {standard_task_runner.py:91} INFO - Job 57: Subtask extract_holiday_events
[2025-08-06T16:04:46.077+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-06T16:04:46.124+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_s3_metadata_dag.extract_holiday_events manual__2025-08-06T16:04:44.805420+00:00 [running]> on host 3bffe12d583b
[2025-08-06T16:04:46.298+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_s3_metadata_dag' AIRFLOW_CTX_TASK_ID='extract_holiday_events' AIRFLOW_CTX_EXECUTION_DATE='2025-08-06T16:04:44.805420+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-06T16:04:44.805420+00:00'
[2025-08-06T16:04:46.300+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-06T16:04:46.329+0000] {base.py:84} INFO - Using connection ID 'mysql_local' for task execution.
[2025-08-06T16:04:46.351+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:243 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-08-06T16:04:46.374+0000] {logging_mixin.py:188} INFO - Extracted holiday_events to /opt/***/temp/holiday_events.csv
[2025-08-06T16:04:46.376+0000] {logging_mixin.py:188} INFO - [LOG] Total row count extracted in the holiday_events is 350
[2025-08-06T16:04:46.376+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/temp/holiday_events.csv
[2025-08-06T16:04:46.376+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-06T16:04:46.395+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_s3_metadata_dag, task_id=extract_holiday_events, run_id=manual__2025-08-06T16:04:44.805420+00:00, execution_date=20250806T160444, start_date=20250806T160445, end_date=20250806T160446
[2025-08-06T16:04:46.431+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-06T16:04:46.444+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
